moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -1051,7 +1051,7 @@ void RobotState::getRobotMarkers(visualization_msgs::msg::MarkerArray& arr, cons
                                  bool include_attached) const
 {
   // ros::Time tm = ros::Time::now();
-
+  rclcpp::Clock clock_ros;
   builtin_interfaces::msg::Time tm = clock_ros.now();
```

Comment:
```suggestion
   builtin_interfaces::msg::Time tm = rclcpp::Clock().now();
```
What about this change?